Dockerfile-cli

```diff
@@ -25,14 +25,14 @@ RUN set -x \
     && docker-php-ext-enable sodium \
     # Removing all PHP leftovers since the helper scripts nor the official image are removing them
     && docker-php-source-tarball clean && rm /usr/local/bin/php-cgi && rm /usr/local/bin/phpdbg && rm -rf /tmp/pear ~/.pearrc \
-    && apk del .phpize-deps
+    && apk del .phpize-deps \
 
-# Patch CVE-2018-14618 (curl), CVE-2018-16842 (libxml2), CVE-2019-11068 (libxslt)
-RUN apk upgrade --no-cache curl libxml2 libxslt
+    # Patch CVE-2018-14618 (curl), CVE-2018-16842 (libxml2), CVE-2019-11068 (libxslt)
```

Comment:
Do we still need to pull in these packages or have the CVE been resolved in more recent alpine versions? Looks like the [oldest one we're building](https://github.com/usabilla/php-docker-template/blob/master/.github/workflows/ci.yml#L165-L176) is `3.10` - [or is it](https://github.com/usabilla/php-docker-template/blob/master/.github/workflows/ci.yml#L24) `3.11`?